src/main/java/org/jabref/model/util/FileHelper.java

```diff
@@ -15,6 +15,15 @@
 public class FileHelper {
 
     /**
+     * Checks if the given String is an online link
+     * @param toCheck The String to check
+     * @return True if it starts with http://, https:// or contains www; false otherwise
+     */
+    public static boolean isOnlineLink(String toCheck) {
+        return toCheck.startsWith("http://") || toCheck.startsWith("https://") || toCheck.contains("www.");
```

Comment:
URL validation is quite hard (potentially completely infeasible) in practice.

Is this implementation sufficient for our purposes? I mean "jabref.org" is a valid url, but would `isOnlineLink()` would return false.

My thoughts are to leave this method in the `LinkedFile` if it should serve as a "special purpose" link checker. If it should be a general method, in the `util` package, then the implementation would need to become more elaborate.

Reply:
I did not change the detection of urls, I just moved it

Reply:
that's what I mean: if you move it to an `util` package, it should be a general purpose method that works in 'all' cases.

If it stays in this class (potentially private) it's sufficient if it is good enough to be called in that specific context.